PORT-13650-add-search-identifiers-to-relevant-guides (#2125)
port-labs/port-docs: docs/guides/all/connect-dynatrace-team-with-entities.md

To set up the mapping, navigate to the Dynatrace integration in the [Data Sources page](https://app.getport.io/settings/data-sources) and add the following mapping based on your preferred method:

<Tabs>
<TabItem value="direct_mapping" label="Direct Mapping" default>

> Direct mapping is when you explicitly specify the identifier of a related entity when creating a relation.   
> This is the traditional way of establishing relations between entities in Port.  
> This is very useful when you're directly referencing the identifier of the related entity

Reviewer: ```suggestion
The most straightforward way to set a relation's value is to explicitly specify the related entity's identifier:
```

:::
</details>

</TabItem>

<TabItem value="search_relation" label="Search Relation">

> You can also use [search relations](https://docs.port.io/build-your-software-catalog/customize-integrations/configure-mapping#mapping-relations-using-search-queries) to dynamically match Dynatrace entities with teams based on specific criteria.   
> This approach is particularly useful when you know the value of one of the entity's properties.  
> 
> Add the snippet below to your mapping configuration to match teams with entities by either using the entity's management zone name or a Kubernetes label.
> You can customize these matching rules according to your organization's team structure and naming conventions.

Reviewer: ```suggestion
You can also use a [search query](https://docs.port.io/build-your-software-catalog/customize-integrations/configure-mapping#mapping-relations-using-search-queries) to dynamically match Dynatrace entities with teams based on specific criteria.   
This approach is particularly useful when you don't know the entity's identifier, but you do know the value of one of its properties.  

Add the snippet below to your mapping configuration to match teams with entities by either using the entity's management zone name or a Kubernetes label.
You can customize these matching rules according to your organization's team structure and naming conventions.
```
